3_visualize/src/plot_gradient_legend.R

```diff
@@ -0,0 +1,98 @@
+plot_gradient_legend <- function(ensemble_data, site_info){
+  
+  # filter data to a focal time period
+  date_start <- as.Date("2021-06-28")
+  
+  threshold = 23.89 # C
```

Comment:
let's pull this out of the plotting function and set it as a config at the top of `_targets.R` to be used across targets